src/test/java/org/mule/runtime/dsl/internal/xml/parser/MuleDocumentLoaderTestCase.java

```diff
@@ -56,13 +46,14 @@ public void tearDown() {
   @Issue("MULE-18813")
   public void documentLoaderShouldNotUseThreadCurrentClassLoader() throws Exception {
 
+    InputStream inputStream = currentThread().getContextClassLoader().getResourceAsStream("simple_application.xml");
     MuleDocumentLoader loader = new MuleDocumentLoader();
 
     // Setting failure class loader
     URLClassLoader failing = mock(URLClassLoader.class);
     currentThread().setContextClassLoader(failing);
 
-    InputSource is = new InputSource(toInputStream(SIMPLE_APPLICATION, UTF_8));
```

Comment:
I assume I'm not affecting the coverage of this test by moving the application data to a file, is that correct?

Having it on a resource file makes it a lot easier to see line and column positions in the other new test are correctly asserted.

Reply:
no, that'sfine